lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -401,19 +406,24 @@ def create_service(self, service_name, cluster_name, task_definition, load_balan
             params['networkConfiguration'] = network_configuration
         if launch_type:
             params['launchType'] = launch_type
+        if self.health_check_setable(params):
+            params['healthCheckGracePeriodSeconds'] = health_check_grace_period_seconds
```

Comment:
I suspect this would pass tests if this only set the value if the parameter wasn't None

Reply:
I guess this was broken by removing the default value as requested by @ezmac 

Will try to add a better check soonish.